lua/telescope/builtin/internal.lua

```diff
@@ -55,6 +62,7 @@ internal.builtin = function(opts)
   }):find()
 end
 
+--- Use the telescope...
 internal.planets = function(opts)
   local show_pluto = opts.show_pluto or false
 
```

Comment:
@tjdevries where everything began 🥰. I suggest moving this to an examples or scratch folder and save space 😬.

Reply:
The planets picker is my most used picker how dare you!!!! You'll have to pry it out of my cold, dead hands :stuck_out_tongue_closed_eyes:

Reply:
🤣🤣🤣🤣. I doubt that @tjdevries would see this anytime soon. So yah let's move it,

Reply:
Don't worry, I'll go pester him on stream later this week until he looks only at this specific comment, and ignores everything else about this PR :sunglasses: :rofl: